docs/library/network.LAN.rst

```diff
@@ -31,13 +31,15 @@ Constructors
        is the default. Suitable values are port specific.
      - *phy_addr* specifies the address of the PHY interface. As with *phy_type*, the hardwired value has
        to be used for most boards and that value is the default.
-     - *phy_clock* specifies, whether the data clock is provided by the Ethernet controller or the PYH interface.
-       The default value is the one that matches the board. If set to ``True``, the clock is driven by the
-       Ethernet controller, otherwise by the PHY interface.
+     - *ref_clk_mode* specifies, whether the data clock is provided by the Ethernet controller or
+       the PYH interface.
+       The default value is the one that matches the board. If set to ``LAN.OUT`` or ``Pin.OUT``
+       or ``True``, the clock is driven by the Ethernet controller, if set to ``LAN.IN``
+       or ``Pin.IN`` or ``False``, the clock is driven by the PHY interface.
 
    For example, with the Seeed Arch Mix board you can  use::
 
-     nic = LAN(0, phy_type=LAN.PHY_LAN8720, phy_addr=2, phy_clock=False)
+     nic = LAN(0, phy_type=LAN.PHY_LAN8720, phy_addr=1, ref_clk_mode=LAN.IN)
```

Comment:
can you please change this example to use `Pin.IN`, because that will be the one which is compatible/consistent with other ports

Reply:
Done.